source/includes/_account.md

```diff
@@ -3904,7 +3904,7 @@ curl -v \
     -H "X-Killbill-CreatedBy: demo" \
     -H "X-Killbill-Reason: demo" \
     -H "X-Killbill-Comment: demo" \
-    -d '{ "accountId": "2ad52f53-85ae-408a-9879-32a7e59dd03d", "pluginName": "__EXTERNAL_PAYMENT__"}' \
+    -d '{ "externalKey": "ExternalKey", "accountId": "2ad52f53-85ae-408a-9879-32a7e59dd03d", "pluginName": "__EXTERNAL_PAYMENT__"}' \
     "http://127.0.0.1:8080/1.0/kb/accounts/8785164f-b5d7-4da1-9495-33f5105e8d80/paymentMethods"	
 ```
```

Comment:
Maybe specify a `Request Body` section (See https://killbill.github.io/slate/#subscription-create-a-subscription) and mention that a `Payment Method Resource` needs to be specified? Also add a link to the [Payment Method Resource](https://killbill.github.io/slate/?shell#payment-method-payment-method-resource) section?

Reply:
Hi @reshmabidikar made this change and added the commit.